src/routes/Home/Home.tsx

```diff
@@ -53,9 +56,37 @@ const Cards = ({ className = '' }: CardsProps) => {
   );
 };
 
+interface CardData {
+  id: string;
+  title: string;
+  description: string;
+  imageSrc: string;
+  imageAlt: string;
+  tags: string[];
+}
+
 const Home = () => {
   const { t } = useTranslation();
   const { sm } = useMediaQueries();
+  const { data } = useLoaderData<LoaderData>();
+  const [newContent, setNewContent] = React.useState<CardData[]>([]);
+
+  React.useEffect(() => {
+    const items = [...data.items].sort(sortContentByDateCreated).map((item) => {
```

Comment:
Sorting is not needed if it is already retrieved correctly sorted from the CMS. See loader cahnges.

Reply:
Used query parameters in the loader, great tip!